just/git.just

```diff
@@ -0,0 +1,19 @@
+import 'default.just'
+
+# Set default values for git info
+GITCOMMIT := env('GITCOMMIT', `git rev-parse HEAD 2> /dev/null || true`)
+GITDATE := env('GITDATE', `git show -s --format='%ct' 2> /dev/null|| true`)
+
+# Find version tag, prioritizing non-rc release tags
+VERSION := env('VERSION', `tags=$(git tag --points-at $GITCOMMIT 2> /dev/null|| true | grep '^op-batcher/' | sed 's/op-batcher\///' | sort -V)
```

Comment:
oops, I missed the tag filtering logic.  Leaving it aside for now just to collect feedback. But this obviously needs to be fixed

Reply:
that's fixed now